cliquet/initialization.py

```diff
@@ -367,6 +367,21 @@ def on_new_response(event):
     config.add_subscriber(on_new_response, NewResponse)
 
 
+def setup_listeners(config):
+    settings = config.get_settings()
+    listeners = [listener.strip() for listener in
+                 settings['event_listeners'].split(',')
+                 if listener.strip()]
+
+    # XXX we should rename it to "RecordChanged" here...
+    from cliquet.events import CollectionChanged
+    #
+    for listener in listeners:
+        listener_mod = config.maybe_dotted(listener)
+        listener = listener_mod.load_from_config(config)
+        config.add_subscriber(listener, CollectionChanged)
```

Comment:
Let's imagine that we don't have the several sections. 

Pyramid multiauth has this pattern of using prefixes to simulate them.  I saw this convention in other pyramid projects too.

> still waiting for feedback on Kinto/kinto#220

Let's say both approaches behave the same way about kwargs being passed from config, it is just a matter of format/syntax IMO